ultimarc/devices/mini_pac.py

```diff
@@ -377,7 +377,7 @@ def _create_message_(self, config_file, cur_device_config=None):
                         pass
 
                 except KeyError:
-                    _logger.debug(_(f'Pin {pin.name} does not exists in Mini-pac device'))
```

Comment:
this name variable shouldn't be changed to product_name.

Reply:
Done.